### PR TITLE
Build question page

### PR DESCRIPTION
**Pull request by ishitakeshawani**
Preview link : https://deploy-preview-3--memorynomads.netlify.app/questionspage/questionspage

### CODE REVIEW

File: Questionspage/questionspage.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    margin: 1rem 0;
+    flex-direction: column;
+  }
+  .question-title {
+    font-size: 1.2rem;
+  }
+  .option {
+    font-size: 0.8rem;
+  }
+  .goback-link,
```
Comment (rohanmathur91) [inline Questionspage/questionspage.css]: take care of spacing between classes

File: index.html [hunk 2]
```diff
@@ -1,63 +1,78 @@
...
+        <h5>Let's play awesome quizzes..</h5>
+        <a href="/Category/category.html"
+          ><button class="btn explore-btn">Explore quizzes</button></a
+        >
+      </div>
+      <img
+        src="assets/home-image.svg"
+        alt="homepage-image"
+        height="400px"
+        width="400px"
```
Comment (rohanmathur91) [inline index.html]: style the image using class

**Review by rohanmathur91 — APPROVED** [review]
Suggested few changes, rest looks good

**Review by prakashsakari — APPROVED** [review]
Looks good to me.

File: Questionspage/questionspage.css (hunk 1 above)
Comment (ishitakeshawani) [reply Questionspage/questionspage.css]: done

**Pull request merged**